eclair-node-gui/src/main/scala/fr/acinq/eclair/gui/GUIUpdater.scala

```diff
@@ -214,15 +214,15 @@ class GUIUpdater(mainController: MainController) extends Actor with ActorLogging
       log.debug(s"payment sent with h=${p.paymentHash}, amount=${p.recipientAmount}, fees=${p.feesPaid}")
       val message = CoinUtils.formatAmountInUnit(p.amountWithFees, FxApp.getUnit, withUnit = true)
       mainController.handlers.notification("Payment Sent", message, NOTIFICATION_SUCCESS)
-      runInGuiThread(() => mainController.paymentSentList.prepend(PaymentSentRecord(p, LocalDateTime.now())))
+      runInGuiThread(() => mainController.paymentSentList.asScala.prepend(PaymentSentRecord(p, LocalDateTime.now())))
```

Comment:
Is the prepend really necessary @dpad85?

Reply:
No it's not necessary and only affects the order of the payments sent (or relayed/received) in the activity tab in the gui. With `prepend`, new events are on top in the lists.

Reply:
Done with 263d96ae1.